### ISSUE TITLE
HoR HC and Normal bugged 

### ISSUE THREAD

**Issue by Echinaceax** [WoWManiaUK/Redemption#3037]
HoR (HC) bugged after 1st boss (Falric – down)  then bugged. 
2nd boss ( Marwyn) didnt spawn

_**Can someone check please**_

- One member of the group had  "quest" (Horde)

**_Edited to add more info_**

(Same group) 
- We tested HoR (normal)

And that bugged on the 9/10 wave

**Comment by Echinaceax** [comment]
Yea im remembering back something about the mage, but i cant remember exactly ..coz i remember thinking huh we killed the mage or so i thought

**Comment by wow-mania** [comment]
Nobody touched it.

**Comment by Dranah** [comment]
Then it simply broke :/
I have thoroughly tested the instance when I fix confirmed the previous thread and I can tell for sure the script has altered since then.

@wow-mania I see some minor improvement but is not full fixed. The mage still casts Phantom Hallucination multiple times below 50% health, instead of being cast only once. (now less frequently recast than before fix)
For the progression issue, I am not sure if it's fixed or i'm failing to reproduce it as we're not fully sure what triggers it bugging in the first place. Will need to push to Red and see if different group setups still get stuck anyhow.

**Comment by Dranah** [comment]
UPDATE:
I have found another issue with outranging during the Escape from LK gauntlet.
If you kill adds rapidly, the distance between sylvanas/jaina and Arthas grows with each wall section and they end up being very far from each other for the last one. So far that you can only have 1 of them in visibility range at a time.
I found that Sylvanas sometimes despawns if you outrange her.
Not sure if this can be helped, I saw similar behavior on escort quests before but for sure it is not intended.

**Comment by wow-mania** [comment]
Fixed allucination issue. I will increase visibility a bit.

**Comment by Dranah** [comment]
This issue still persists.
Phantom Hallucination is recast multiple times after the mage returns from Invisibility.
Doing the LK escape too fast (sylv/jaina outranging arthas) still ends the event, but this time, LK will cast Fury of the Frostmourne and obliterate your party.
Steps to repro:
- Stay near LK and kill all spawning adds asap
- After 3rd wall, sylv/jaina will be too far from arthas. The NPC is insta-killed by an unknown source, LK casts Fury of the Frostmourne and you have to restart event.

**Comment by wow-mania** [comment]
Usually, you don't stay near LK you objective is to keep Jaina safe so I would not say this is standard behavior.

**Comment by wow-mania** [comment]
@Keader can you take over this one also please.

**Comment by Dranah** [comment]
@wow-mania It does not only happen if you stay with LK, I have had reports of players from Redemption who did the regular way and it still happened. I can just reproduce it more steadily this way.
I am also seeing the other bug lately, which is that sometimes the wall won't fall when the last add in the wave dies, there are no adds up but it simply isn't letting you move on. You can do nothing but wait for LK to come and kill the NPC and wipe your party. This one seems to happen if you're staying with jaina/Sylv.

The most direct issue is however the Phantom hallucinations. Basically once the mage gets below 50%, she keeps spawning them unstoppably and if you don't finish her off quickly enough, 3-4 hallucinations can easily cull your group, especially if the tank eats a CC.
This one needs to be fixed.
Something has changed on this instance roughly 8-9days ago and i'd be really curious as to what it was, since it worked flawlessly after we closed my last report. If we could find what screwed it up and revert it, it could solve all the issues at once...

**Comment by Keader** [comment]
Fixed.
About LK, will not cast fury anymore... but kill monster so quickly may can cause weird behavior anyway...
